dask_ml/wrappers.py

```diff
@@ -386,7 +386,7 @@ class Incremental(ParallelPostFit):
     used during the call to ``fit``. All attributes learned during training
     are available on ``Incremental`` directly.
 
-    .. _list of incremental learners: http://scikit-learn.org/stable/modules/scaling_strategies.html#incremental-learning  # noqa
+    .. _list of incremental learners: https://scikit-learn.org/0.15/modules/scaling_strategies.html#incremental-learning  # noqa
```

Comment:
I think this page was moved to https://scikit-learn.org/stable/modules/computing.html#incremental-learning

Reply:
@jrbourbeau are the changes okay ? or do I need to update the link that you mentioned!

Reply:
Preferably update to @jrbourbeau's link, so that as changes are made they're reflected in what we link to.

Reply:
Sure @TomAugspurger . I will modify it and send a PR